doc/examples/streamline_registration.py

```diff
@@ -19,8 +19,13 @@
 first, by importing it. This provides us with all of the variables that were
 created in that example.
 
-In order to get the deformation field, we will first use two b0 volumes. The
-first one will be the b0 from the Stanford HARDI dataset:
+In order to get the deformation field, we will first generate a mean b0 volume
+(to restrict brain boundaries) and an FA map (to yield a high-contrast wm tissue
+image that is ideal for optimizing subcortical registration). To accomplish
+this, we will use the Stanford HARDI dataset, and an mni-space FA template from
```

Comment:
Telling where the streamlines come from is something that I also missed (and it is still not clear to the reader in the example), so I'll add a sentence stating that.

Reply:
OK. Somehow I got confused; this was already explained in the upstream version.